src/components/SortBar.jsx

```diff
@@ -0,0 +1,36 @@
+export default function SortBar () {
+
+    return (
+        <nav className="nav-sort">
+            <form action={'/articles'}>
```

Comment:
this is causing a complete refresh of your app 

In SinglePageApplications you can provide a better user experience by responding to selects with state updates (or by using the `useSearchParams` hook to add queries to the URL programmatically!)